edkrepo/common/clone_utilities.py

```diff
@@ -57,6 +67,25 @@ def generate_clone_cmd(repo_to_clone, workspace_dir, args=None, cache_path=None)
         except AttributeError:
             pass
 
+    try:
+        if repo_to_clone.treeless:
+            clone_cmd_args['treeless repo'] = '--filter=tree:0'
+    except AttributeError:
+        pass
+    try:
+        if repo_to_clone.blobless:
+            clone_cmd_args['blobless repo'] = '--filter=blob:none'
+    except AttributeError:
+        pass
+
+    if len(active_filters) > 1:
+        raise edkrepo_exception.EdkrepoInvalidConfigOptionException(humble.CONFLICTING_PARTIAL_CLONE)
+    elif len(active_filters) == 1:

```

Comment:
This elif clause functionally does nothing since we are just adding two additional spaces to the clone command.

Reply:
To avoid confusion, 
will change 'treeless repo' to 'treeless_manifest_setting'
will change 'blobless repo' to 'blobless_manifest_setting'